PowerEditor/src/ScitillaComponent/UserDefineDialog.h

```diff
@@ -417,12 +421,20 @@ public :
 protected :
     INT_PTR CALLBACK run_dlgProc(UINT Message, WPARAM wParam, LPARAM);
 
+	// Custom proc to subclass edit control
+	LRESULT static CALLBACK customEditProc(HWND hEdit, UINT msg, WPARAM wParam, LPARAM lParam);
+
+	bool isAllowed(const generic_string& txt);
+	void HandlePaste(HWND hEdit);
+
 private :
     generic_string _title;
     generic_string _textValue;
     generic_string _static;
+	generic_string _restrictedChars;

```

Comment:
extra tab?

Reply:
No.  Moreover, it's not me. Visual Studio does it automatically.

`    generic_string _static;` => There are 4 spaces
`	generic_string _restrictedChars;` => Here it is a tab.
![image](https://user-images.githubusercontent.com/14791461/53697731-504f3f80-3dfa-11e9-9fb5-85bad6be86e1.png)

Reply:
@SinghRajenM:  I usually solve this issue by highlighting the "tab" character, typing `Ctrl`+`H` (find and replace), then entering 4 spaces into the "replace" field.